Makefile

```diff
@@ -185,7 +185,7 @@ crds: $(CONTROLLER_GEN) $(YQ)
 
 gen-rbac: $(HELM) $(YQ) ## generate RBAC from Helm charts
```

Comment:
`gen-rbac` is run as part of the `make build`, right?

Reply:
No. It is separate currently.